changelog.md

```diff
@@ -4,20 +4,116 @@ All notable end-user facing changes are documented in this file.
+  - Updated Google.Protobuf dependency #2096 
+  - Service Provider is no longer required by EventHubAdapter #2044
+  - Preliminary relational persistence queries #1682
+  - Add a function that checks the connection string for use during initialization #1987
+  - Updated Azure Storage dependency to 7.0.0 #1968
+  - Added new Amazon AWS basic Orleans providers [#2006](https://github.com/dotnet/orleans/issues/2006)
+  - A new ADO.NET storage provider that is significantly easier to setup, which replaces the the previous one. This change is not backwards compatible and does not support sharding
   (likely be replaced later with Orleans sharding provider). The most straightforward migration plan is likely to persist the state classes from Orleans application code.
-  More information in [#1682](https://github.com/dotnet/orleans/pull/1682) and in [#1682 (comment)](https://github.com/dotnet/orleans/pull/1682#issuecomment-234371701). 
-- Added new Amazon AWS basic Orleans providers [#2006](https://github.com/dotnet/orleans/issues/2006)
-- Moved Orleans Performance Counters into its own Telemetry Consumer. Now you need to explicitly register the `OrleansPerfCounterTelemetryConsumer` either by code or XML. More information in [#2122](https://github.com/dotnet/orleans/pull/2122) and docs will come later
-	- `Microsoft.Orleans.CounterControl` can still be used to install the performance counters or you can use `InstallUtil.exe OrleansTelemetryConsumers.Counters.dll` to install it without depending on `OrleansCounterControl.exe`
+  More information in [#1682](https://github.com/dotnet/orleans/pull/1682) and in [#1682 (comment)](https://github.com/dotnet/orleans/pull/1682#issuecomment-234371701).
+  - Update ZooKeeperNetEx to 3.4.8.5 #1927
+  - Memory Storage eTag enforcement less strict. #1885
+  - Relational Storage: Added SiloName to Membership table #1787
+  - Added option to perform provider commands quietly #1762
+  - Fix for strange bug in BondSerializer #1790
+  - CreateOrleansTables_SqlServer.sql: Removed support for SQL Server 2000 and 2005 #1779
+  - Added support for using Consul 0.6.0 as a Membership Provider #1267
+  - Allow for extraneous keys in OrleanQuery table #1752
+- Streaming
+  - EventHub receiver metrics reporting abstraction #2146
+  - EventHub stream provider time based message purge #2093
+  - Upgrade to ServiceBus 3.3.2 and related changes. #2087
+  - Add Memory Stream Provider #2063
+  - Persistent stream pulling agent now uses exponential backoff #2078
+  - Add dynamic adding / removing stream providers functionality. #1966
+  - EventHub stream provider logging/monitoring #1857
+  - Updating EventHubAdapterFactory for extensibility. #1861
+  - Consistent implicit subscription Id generation. #1828
+  - ServiceBus upgrade to 3.2.2 #1813
+  - Event hub stream provider EventData to cached data mapping #1727
+  - Refactor of EventHubStreamProvider to better support plugable CacheDataAdapter #1714
```

Comment:
Several EventHub provider refactoring and making it extensible that should probably be summarized into 1 item only